src/components/fields/money-field/money-field.spec.js

```diff
@@ -115,13 +112,20 @@ it('should call onBlur when amount input loses focus', () => {
 it('should call onBlur when currency select loses focus', () => {
   const onBlur = jest.fn();
   const { getByLabelText } = renderMoneyField({ onBlur });
-  getByLabelText('Currency Code').focus();
-  expect(getByLabelText('Currency Code')).toHaveFocus();
-  getByLabelText('Currency Code').blur();
-  expect(getByLabelText('Currency Code')).not.toHaveFocus();
+  getByLabelText('EUR').focus();
+  expect(getByLabelText('EUR')).toHaveFocus();
+  getByLabelText('EUR').blur();
+  expect(getByLabelText('EUR')).not.toHaveFocus();
   expect(onBlur).toHaveBeenCalled();
 });
 
+it('should have focus the amount input automatically when isAutofocussed is passed', () => {
```

Comment:
should have focussed? or should focus the?

Reply:
🙈